_data/simple-icons.json

```
@@ -8117,6 +8117,11 @@
             "hex": "F43E37",
             "source": "https://blog.pocketcasts.com/press/"
         },
+        {
+            "title": "PocketBase",
+            "hex": "B8DBE4",
+            "source": "https://pocketbase.io"
```

```suggestion
            "source": "https://github.com/pocketbase/pocketbase/blob/4b64e0910b7dc527ff3de8cdacec074e40449e2e/ui/dist/images/logo.svg"
```

Let's use the permalink instead.